nx/lib/nx/lin_alg.ex

```diff
@@ -277,11 +277,25 @@ defmodule Nx.LinAlg do
   end
 
   @doc """
-  Solve the equation `a x = b` for x, assuming `a` is a lower triangular matrix.
+  Solve the equation `a x = b` for x, assuming `a` is a triangular matrix.
+  Can also solve `x a = b` for x. See the `:left_side` option below.
 
   `b` must either be a square matrix with the same dimensions as `a` or a 1-D tensor
   with as many rows as `a`.
 
+  ## Options
+
+  The following options are defined in order of precedence
+
+  * `:transform_a` - Defines `op(a)`, depending on its value. Can be one of:
+    * `:none` -> `op(a) = a`
+    * `:transpose` -> `op(a) = transpose(a)`
```

Comment:
In my opinion we don't need to support transpose either, but since you added it, we can keep it. :)